src/utils.cpp

```diff
@@ -96,6 +96,11 @@ int SmallestPowerOf2(int x) {
 	return x;
 }
 
+bool IsCharRTL(wxChar c) {
+	// only hebrew for now
+	return (((wxChar)0x5d0) <= c) && (c <= ((wxChar)0x5ea));
```

Comment:
I haven't tested, but this should be better and support all the RTL character.
```suggestion
bool IsCharRTL(wxChar character) {
    hb_unicode_funcs_t* unicode_funcs = hb_unicode_funcs_get_default();
    hb_script_t script = hb_unicode_script(unicode_funcs, character);
    return hb_script_get_horizontal_direction(script) == HB_DIRECTION_RTL;
}
```